packages/api-server/src/api/admin.ts

```diff
@@ -1,60 +1,53 @@
-import { Request, Response, Router } from 'express'
+import { Router } from 'express'
 
 import { db, redis } from '../db'
-import wrapAsync from '../middleware/wrapAsync'
 
 const router = Router()
 
-router.get(
-  '/listServers',
-  wrapAsync(async (req: Request, res: Response) => {
-    try {
-      const dbServerRes = await db.dbQuery(
-        `SELECT * FROM servers ORDER BY id`,
-        []
-      )
-
-      const servers = dbServerRes.rows.map((s) => ({
-        id: s.id,
-        discordId: s.discord_id,
-        discordName: s.discord_name,
-        subdomain: s.subdomain,
-        createdOn: s.created_on,
-      }))
-
-      return res.status(200).json(servers)
-    } catch (err) {
-      return res.status(500).write(err)
+router.get('/listServers', async (req, res) => {
```

Comment:
Removing Request/Response here doesn't mess up TS linting or VSCode or anything does it? 

I would think not, but just making sure.